buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -110,7 +110,7 @@ object Libraries {
     const val accompanistNavAnimation   = "com.google.accompanist:accompanist-navigation-animation:${Versions.accompanist}"
     const val accompanistIndicator      = "com.google.accompanist:accompanist-pager-indicators:${Versions.accompanist}"
     const val material                  = "com.google.android.material:material:${Versions.material}"
-    const val visibilityModifiers       = "io.github.esentsov:kotlin-visibility:${Versions.visibilityModifiers}"
+    const val visibilityModifiers        = "io.github.esentsov:kotlin-visibility:${Versions.visibilityModifiers}"
```

Comment:
```suggestion
    const val visibilityModifiers       = "io.github.esentsov:kotlin-visibility:${Versions.visibilityModifiers}"
```

Reply:
This is how it looks to me now

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/9389043/184370039-a2da2918-e2de-4c35-ab85-eb8114abe6f9.png">

What is this wizzardry?

Reply:
![image](https://user-images.githubusercontent.com/9083456/184376586-4ea68910-9c49-405b-9d2e-50142eba44de.png)

Reply:
Yes, on GitHub it looks weird. On the IDE it looks aligned.